### PR TITLE
Implement cache directive value

### PR DESCRIPTION
**Pull request by tetsuharuohzeki**
This implements value definitions and simple value builder for `Cache-Control` http header values.

## Spec/Related Links

- https://www.rfc-editor.org/rfc/rfc9111.html#name-request-directives
- https://www.rfc-editor.org/rfc/rfc9111.html#name-response-directives
- https://datatracker.ietf.org/doc/rfc5861/
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control

### CODE REVIEW

File: packages/http-helper/src/cache/request_directive.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-no-transform
+ */
+export const NO_TRANSFORM = 'no-transform' as const;
+
+/**
+ *  - https://www.rfc-editor.org/rfc/rfc9111.html#name-only-if-cached
+ */
+export const ONLY_IF_CACHED = 'only-if-cached' as const;
```
Comment (negibokken) [inline packages/http-helper/src/cache/request_directive.ts:38]: According to the RFC and MDN, it seems `stale-if-error` can come to request.

https://datatracker.ietf.org/doc/html/rfc5861#section-4
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Cache-Control#cache_directives

**Review by negibokken — COMMENTED** [review]
Could you add `stale-if-error` in the request directive?

**Comment by tetsuharuohzeki** [comment]
I pushed https://github.com/Nikkei/node-http-helper/pull/193/commits/f64fb96f88cee674938d06ba74003e98fbb43883 to improve the result of `buildCacheControlValue()` more.

File: packages/http-helper/src/cache/request_directive.ts (hunk 1 above)
Comment (tetsuharuohzeki) [reply packages/http-helper/src/cache/request_directive.ts:38]: addressed:

- https://github.com/Nikkei/node-http-helper/pull/193/commits/0fcfa4be895848021d09a7d8c1e85d740d7fdff8
- https://github.com/Nikkei/node-http-helper/pull/193/commits/5679941e67d2be1cb550dd608383005e3a562756

**Review by negibokken — APPROVED** [review]
LGTM! 👀

**Comment by tetsuharuohzeki** [comment]
@negibokken Thanks!

**Pull request merged**